gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -428,8 +428,7 @@ class {{service.name}}RestTransport({{service.name}}Transport):
             {% endif %}{# method.void #}
      {% else %}{# method.http_options and not method.client_streaming #}
             {% if not method.http_options %}
-            raise RuntimeError(
-                "Cannot define a method without a valid 'google.api.http' annotation.")
+            raise NotImplementedError("This method does not support REST transport.")
```

Comment:
On my local branch, I was simplifying this whole else clause to also account for the streaming error. I think that's worth doing:
```
     {% else %}{# method.http_options and not method.client_streaming #}
            raise NotImplementedError(
                "Method {{ method.name }} is not available over REST transport")

    {%- endif %}{# unary method #}

```

Reply:
Done.